dictionary/lib/dictionary/supervisor.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Dictionary.Supervisor do
+
```

Comment:
So you've had to duplicate @default name in two modules.

Can you think of a way of eliminating that?

Reply:
good point - I was thinking of using "use" or "import" somehow?  Will look a bit into this.